.github/workflows/nightly_release.yml

```diff
@@ -1,24 +1,17 @@
 name: Push Nightly Release
 
 on:
-  # [ Note: Manually Trigger the Workflow ]
-  # 1. Go to Actions under pytorch/data repo
-  # 2. In the left sidebar, click the workflow you want to run
-  # 3. Above the list of workflow runs, select Run workflow
-  # 4. Use the Branch dropdown to select the release/* branch
-  # 5. Click Run workflow
-  workflow_dispatch:
-  # TODO: Automatically trigger nightly release
-  # schedule:
-  #   - cron: 30 23 * * *
+  schedule:
+    - cron: 30 23 * * *
```

Comment:
Turn on nightly release. 

For Doc generation, I think there are two options for us:
- Create another folder called `nightly` just for nightly release
- Disable the doc generation workflow that we are using in main branch, and enable the doc gen workflow whenever a nightly release is uploaded.

cc: @NivekT

Reply:
I think the second one (only generate doc for nightly) is fine. We likely don't need to re-generate the doc for every single commit to main.

Reply:
Sounds great. Will wait until the nightly works.